ui/src/onboarding/containers/LoginPage.scss

```diff
@@ -0,0 +1,168 @@
+.clockface--app-wrapper.sign-up--page {
+  background-image: linear-gradient(
+    47deg,
+    rgba(19, 0, 45, 0) 65%,
+    rgba(191, 47, 229, 0.4) 100%
+  ) !important;
+  background-color: #13002d !important;
+}
+
+.sign-up--page-contents {
+  .cf-page-contents--padding {
+    height: 100vh;
+    display: inline-flex;
+    width: 100%;
+  }
+}
+
+.sign-up--form {
+  min-width: 330px;
+  width: 100%;
+  height: 100%;
+  display: inline-flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  .sign-up--form-padded-row {
+    padding-top: $ix-marg-d;
+  }
+}
+
+.sign-up--login-container {
+  padding: $ix-marg-b;
+  text-align: center;
+  color: $g20-white;
+}
+
+.sign-up--login-text {
+  font-size: 16px;
+  margin-bottom: 0;
+}
+
+.sign-up--form-panel {
+  max-width: 500px;
+  width: 400px;
+  height: 100%;
+  justify-content: space-between;
+}
+
+.sign-up--panel {
+  height: 100%;
+  max-width: 1600px;
+  min-height: 670px + $ix-marg-d;
+  padding: $ix-marg-d;
+  margin: auto auto;
+  z-index: 0;
+  background: transparent !important;
+  z-index: 100;
+}
+
+@media screen and (min-width: $grid--breakpoint-md) {
+  .sign-up--form-panel {
+    max-width: 100%;
+  }
+
+  .sign-up--full-height {
+    min-height: 100%;
+  }
+
+  .sign-up--header-align {
+    text-align: center;
+  }
+
+  .sign-up--panel:not(.aws) {
+    height: 80vh;
+    padding: 64px;
+  }
+}
+
+@media screen and (min-width: $grid--breakpoint-md) {
```

Comment:
i think you can cut this and fold these two class selectors up into the above media declaration, since it does the same thing - `@media screen` and `min-width: $grid--breakpoint-md`